samples/wifi/radio_test/src/nrf_wifi_radio_ficr_shell.c

```diff
@@ -226,14 +227,16 @@ static int nrf_wifi_radio_test_otp_write_params(const struct shell *shell,
 		write_val[2]  = strtoul(argv[4], NULL, 0);
 		write_val[3]  = strtoul(argv[5], NULL, 0);
 		/* All consecutive 4 words of the qspi keys are written now */
-		write_otp_memory(QSPI_KEY, &write_val[0]);
+		status = write_otp_memory(QSPI_KEY, &write_val[0]);
 		break;
 	default:
 		shell_fprintf(shell, SHELL_ERROR, "unsupported field %d\n", field);
 		return -ENOEXEC;
 	}
 
-	shell_fprintf(shell, SHELL_INFO, "Finished Writing OTP params\n");
+	if (!status) {
+		shell_fprintf(shell, SHELL_INFO, "Finished Writing OTP params\n");
+	}
```

Comment:
A overall failure statement in else or are you relying on failure logs from above?

Reply:
The print here will be misleading if the logs preceding this indicate that there is an error in OTP update.